helmfile/values/velero-sc.yaml.gotmpl

```diff
@@ -8,14 +8,14 @@ nodeSelector: {{- toYaml .Values.velero.nodeSelector | nindent 2  }}
 initContainers:
   {{- if eq .Values.objectStorage.type "s3" }}
   - name: velero-plugin-for-aws
-    image: velero/velero-plugin-for-aws:v1.3.1
+    image: velero/velero-plugin-for-aws:v1.5.1
     imagePullPolicy: IfNotPresent
     volumeMounts:
       - mountPath: /target
         name: plugins
   {{- else if eq .Values.objectStorage.type "gcs" }}
   - name: velero-plugin-for-gcs
-    image: velero/velero-plugin-for-gcp:v1.1.0
+    image: velero/velero-plugin-for-gcp:v1.5.1
```

Comment:
Did you check that this version exists?
I mean the versions weren't the same before for gcp and aws ;)

Reply:
yes I did check